analysis/tools/ccsh/src/main/kotlin/de/maibornwolff/codecharta/tools/ccsh/parser/InteractiveParserSuggestionDialog.kt

```diff
@@ -21,12 +21,14 @@ class InteractiveParserSuggestionDialog {
                 return emptyMap()
             }
 
+            val parserRepository = PicocliParserRepository()
```

Comment:
The repository object is used only in the else right? 
So let's move this line down into the else.